homeassistant/components/light/greenwave.py

```diff
@@ -27,14 +27,14 @@
     vol.Required(CONF_VERSION): cv.positive_int,
 })
 
-MIN_TIME_BETWEEN_UPDATES = timedelta(minutes=1)
+MIN_TIME_BETWEEN_UPDATES = timedelta(seconds=5)
```

Comment:
For polling data over TCP this is very fast as there is no session used.

See https://github.com/dfiel/greenwavereality/issues/1 for the request to add a timeout.